docker/bin/entrypoint.sh

```diff
@@ -8,12 +8,19 @@ if [ -f ${HASSIO_OPTIONSFILE} ]; then
     CONFIG=$(grep config_file ${HASSIO_OPTIONSFILE}| cut -d ':' -f 2 | sed s/[\"}]//g )
     echo "Using config file: ${CONFIG}"
 
-    if [ ! -f ${CONFIG} ]; then
+    SQLITE_FILE=$(grep -o '"sqlite_file": "[^"]*' ${HASSIO_OPTIONSFILE} | grep -o '[^"]*$')
+
+    if [ ! -f "${CONFIG}" ]; then
         echo "Config not found. Please create a config under ${CONFIG}."
         echo "For details see evcc documentation at https://github.com/evcc-io/evcc#readme."
     else
-        echo "starting evcc: 'evcc --config ${CONFIG}'"
-        exec evcc --config ${CONFIG}
+        if [ -z "${SQLITE_FILE}" ]; then
```

Comment:
Why not checking if SQLITE_FILE is valid and then using it? Currently we can not add further options in future. I would suggest. empty use config file. if not empty check against possible options like sql…

Reply:
Do we have grep installed?

Reply:
@Hofyyy elaborate please... do you want me to check if SQLITE_FILE _is not empty_ ? So simply swapping the if-else-clause?

@andig we have, see Line 8, initialisation of CONFIG. But instead of doing grep-cut-sed magic i'm just using two greps with RegEx's